flink-runtime-web/src/main/java/org/apache/flink/runtime/webmonitor/RuntimeMonitorHandler.java

```diff
@@ -113,7 +117,17 @@ private void respondAsLeader(ChannelHandlerContext ctx, Routed routed, ActorGate
 		DefaultFullHttpResponse response;
 
 		try {
-			String result = handler.handleRequest(routed.pathParams(), jobManager);
+			Map<String, String> params = routed.pathParams();
```

Comment:
I see, that is a case for query parameters. Is it better to have them all in one map, or to simply give the function two maps?